frontend/ui/data/onetime_migration.lua

```diff
@@ -34,15 +34,6 @@ local function drop_fontcache()
     end
 end
 
--- Global settings, https://github.com/koreader/koreader/pull/4945 & https://github.com/koreader/koreader/pull/5655
--- Limit the check to the most recent update. ReaderUI calls this one unconditionally to update docsettings, too.
-if last_migration_date < 20191129 then
-    logger.info("Performing one-time migration for 20191129")
-
-    local SettingsMigration = require("ui/data/settings_migration")
-    SettingsMigration:migrateSettings(G_reader_settings)
```

Comment:
This one has not been used since 2019 - not sure why. May be it existed before we introduced onetimemigration, which is where migration stuff has been happening :)  @NiLuJe ?
So, dunno if it's a good idea to ressurect/reuse that one, which did/would do other stuff.

Reply:
No strong feeling about it, but it probably couldn't hurt to keep the old one as-is, and just add another specific method in `SettingsMigration` for that one ;).

Reply:
onetimemigration also handles global settings. It was introduced by you in April 2021.
settings_migration has had minor stuff added after 2019, but given that it's driven since Apr 2021 by your PR with a last_migration_date < 2019 condition, it has had no effect and use.
I'd be for keeping everything in onetime_migration.